src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -472,6 +478,21 @@ protected GenomicsDBOptions getGenomicsDBOptions() {
     public void onTraversalStart() {
         final Map<String, VCFHeader> vcfHeaders = Collections.singletonMap(getDrivingVariantsFeatureInput().getName(), getHeaderForVariants());
 
+        final List<String> genotypeField = getHeaderForVariants().getGenotypeSamples();
```

Comment:
This is from the wrong PR. #7887. Can you rebase and remove the `SelectVariants` commit please?